### PR TITLE
change pi groups, add Deardorff velocity to SurfaceBase

### PR DESCRIPTION
**Pull request by costachris**
Modify pi groups to remain O(1), add Deardorff velocity to `SurfaceBase`

### CODE REVIEW

File: src/closures/nondimensional_exchange_functions.jl [hunk 1]
```diff
@@ -10,9 +10,9 @@ end
 function non_dimensional_groups(param_set, εδ_model_vars)
     Δw = get_Δw(param_set, εδ_model_vars.w_up, εδ_model_vars.w_en)
     Δb = εδ_model_vars.b_up - εδ_model_vars.b_en
-    FT = eltype(εδ_model_vars.tke)
-    Π_1 = (εδ_model_vars.zc_i * Δb) / Δw^2
-    Π_2 = Δw^2 / (εδ_model_vars.tke + eps(FT)) # prevent division by 0
+    FT = eltype(εδ_model_vars.tke_en)
+    Π_1 = (εδ_model_vars.zc_i * Δb) / (Δw^2 + εδ_model_vars.wstar^2)
+    Π_2 = (εδ_model_vars.tke_gm - εδ_model_vars.a_up * εδ_model_vars.tke_en) / (εδ_model_vars.tke_en + eps(FT))
```
Comment (costachris) [inline src/closures/nondimensional_exchange_functions.jl:15]: After discussions with @yairchn , this is equivalent to the suggested `a(1-a)dw^2/gm_TKE`, but works for multiple updrafts.

Comment (ilopezgp) [inline src/closures/nondimensional_exchange_functions.jl:15]:
```suggestion
    Π_2 = (εδ_model_vars.tke_gm - εδ_model_vars.a_up * εδ_model_vars.tke_en) / (εδ_model_vars.tke_gm + eps(FT))
```

Comment (yairchn) [reply src/closures/nondimensional_exchange_functions.jl:15]: it should be 
'Π_2 = (εδ_model_vars.tke_gm - εδ_model_vars.a_en * εδ_model_vars.tke_en) / (εδ_model_vars.tke_gm + eps(FT))'

Comment (yairchn) [reply src/closures/nondimensional_exchange_functions.jl:15]: in the numerator you should have env area multiplying the env TKE

**Review by ilopezgp — APPROVED** [review]
LGTM

**Review by yairchn — APPROVED** [review]
great work

File: src/closures/nondimensional_exchange_functions.jl [hunk 2]
```diff
@@ -10,9 +10,10 @@ end
 function non_dimensional_groups(param_set, εδ_model_vars)
     Δw = get_Δw(param_set, εδ_model_vars.w_up, εδ_model_vars.w_en)
     Δb = εδ_model_vars.b_up - εδ_model_vars.b_en
-    FT = eltype(εδ_model_vars.tke)
-    Π_1 = (εδ_model_vars.zc_i * Δb) / Δw^2
-    Π_2 = Δw^2 / (εδ_model_vars.tke + eps(FT)) # prevent division by 0
+    FT = eltype(εδ_model_vars.tke_en)
+    Π_1 = (εδ_model_vars.zc_i * Δb) / (Δw^2 + εδ_model_vars.wstar^2)
+    Π_1 = Π_1 / 478.298 # Normalize by max(abs(Π_1)) found in {Bomex, DYCOMS, TRMM}
```
Comment (charleskawczynski) [inline src/closures/nondimensional_exchange_functions.jl:15]: It's fine for now, but maybe later we can wrap the `478.298` with `FT` to preserve type stability

**Comment by costachris** [comment]
> Build failed:
> 
> * [buildkite/turbulenceconvection-ci](https://buildkite.com/clima/turbulenceconvection-ci/builds/8464)

SAD

**Comment by charleskawczynski** [comment]
:frowning_face:

**Comment by costachris** [comment]
Sorry I was adding the `FT`

**Pull request merged**